tableauserverclient/server/server.py

```diff
@@ -41,6 +41,9 @@ def __init__(self, server_address):
         self.schedules = Schedules(self)
         self.server_info = ServerInfo(self)
 
+        if use_server_version:
+            self.use_highest_version()
```

Comment:
This will throw the warning every time someone uses the flag since this is the legacy endpoint.

I think you want this to be `use_server_version` too :)